linux/buildScript.sh

```diff
@@ -11,12 +11,12 @@ set -u
 
 srctgz="https://www.census.gov/ts/x13as/unix/x13ashtmlsrc_V1.1_B19.tar.gz"
 file=$(basename ${srctgz})
-
 td=$(tempfile -p x13)
+rm -r $td
```

Comment:
@christophsax and I apologize for the weird directory layout. The 'os-at-top-level' scripts where here first, we later decided a 'version-at-top-level' was better to provide multiple versions. So please take what is in `v1.1.26` as more relevant.

And let me just say that we _do_ appreciate the interest and help.  So far, use had been limited to the 'big three' with Linux being x64 mostly.  This extends the reach.